docs/UserGuide.md

```diff
@@ -152,12 +152,12 @@ Adds a user-defined anime into the anime list.
 Format: `add n/NAME [e/EPISODE] [s/STATUS] [g/GENRE] [g/GENRE] ...`
 
 Notes:
-* If multiple `n/NAME` are given in the command, the last provided `NAME` will be used
-* If multiple `e/EPISODE` are given in the command, the last provided `EPISODE` will be used
-* If multiple `s/STATUS` are given in the command, the last provided `STATUS` will be used
+* If more than one `n/NAME` is given in the command, the last provided `NAME` will be used
+* If more than one `e/EPISODE` is given in the command, the last provided `EPISODE` will be used
+* If more than one `s/STATUS` is given in the command, the last provided `STATUS` will be used
```

Comment:
why change from multiple to more than one tho, multiple is just one word and conveys the same meaning, making it more neat

Reply:
so it doesnt look like we have grammar issues. originally it is multiple s/STATUS (SINGULAR). we can circumvent this problem by using `more than one`